packages/components/pie-webc-testing/src/helpers/set-rtl-direction.ts

```diff
@@ -0,0 +1,7 @@
+import { Page } from '@playwright/test';
```

Comment:
I don't think it makes a difference in test code, but just as a good habit you should be able to import this as a type rather than a value:
```suggestion
import { type Page } from '@playwright/test';
```